_recipes/lists-by-number-of-items.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Lists by number of items
+layout: site
+body_class: recipe
+keywords: reading lists, items, list analytics
+description: Use reading intentions to show whether students are interacting with lists. Find out which students are interacting with which lists for use in retention metrics.
```

Comment:
This description needs updating. Couple of sentences that sum up the broad area addressed by this article.